Index: b/configure
===================================================================
--- a/configure
+++ b/configure
@@ -554,8 +554,8 @@ fi
 
 if test "$f_libclang" = YES; then
   printf "  Checking for libclang ... "
-  libclang_hdr_dir="/usr/include /usr/local/include /opt/local/include"
-  libclang_lib_dir="/usr/lib /usr/local/lib /opt/local/lib /usr/lib64/llvm /usr/lib/llvm"
+  libclang_hdr_dir="/usr/lib/llvm-3.5/include /usr/include /usr/local/include /opt/local/include"
+  libclang_lib_dir="/usr/lib/llvm-3.5/lib /usr/lib /usr/local/lib /opt/local/lib /usr/lib64/llvm /usr/lib/llvm"
   if test "$f_libclangstatic" = NO; then
     libclang_lib_name="libclang.so libclang.dylib libclang.a libclang.dll.a"
   else
Index: b/src/libdoxygen.pro.in
===================================================================
--- a/src/libdoxygen.pro.in
+++ b/src/libdoxygen.pro.in
@@ -230,6 +230,7 @@ win32-g++:TMAKE_CXXFLAGS   += -fno-excep
 linux-g++:TMAKE_CXXFLAGS   += -fno-exceptions
 INCLUDEPATH                += ../generated_src/doxygen ../src ../qtools ../libmd5 ../vhdlparser
 INCLUDEPATH                += %%SQLITE3_INC%%
+INCLUDEPATH                += /usr/lib/llvm-3.5/include
 DEPENDPATH                 += ../generated_src/doxygen
 win32:INCLUDEPATH          += .
 DESTDIR                    =  ../lib
