Index: doxygen-1.7.6.1/doc/config.doc
===================================================================
--- doxygen-1.7.6.1.orig/doc/config.doc	2011-12-06 21:07:10.000000000 +0000
+++ doxygen-1.7.6.1/doc/config.doc	2011-12-12 14:30:30.457971073 +0000
@@ -2321,10 +2321,10 @@
 <dt>\c DOT_NUM_THREADS <dd>
  \addindex DOT_NUM_THREADS
  The \c DOT_NUM_THREADS specifies the number of dot invocations doxygen is 
- allowed to run in parallel. When set to 0 (the default) doxygen will 
+ allowed to run in parallel. When set to 0 (the upstream default) doxygen will 
  base this on the number of processors available in the system. You can set it 
  explicitly to a value larger than 0 to get control over the balance 
- between CPU load and processing speed.  
+ between CPU load and processing speed.  For Debian the default is 1.
 
 \anchor cfg_dot_fontname
 <dt>\c DOT_FONTNAME <dd>
Index: doxygen-1.7.6.1/src/config.xml
===================================================================
--- doxygen-1.7.6.1.orig/src/config.xml	2011-12-06 21:03:24.000000000 +0000
+++ doxygen-1.7.6.1/src/config.xml	2011-12-12 14:29:03.277842085 +0000
@@ -1446,11 +1446,11 @@
 ' defval='0'/>
     <option type='int' id='DOT_NUM_THREADS' docs='
 The DOT_NUM_THREADS specifies the number of dot invocations doxygen is 
-allowed to run in parallel. When set to 0 (the default) doxygen will 
+allowed to run in parallel. When set to 0 (the upstream default) doxygen will 
 base this on the number of processors available in the system. You can set it 
 explicitly to a value larger than 0 to get control over the balance 
-between CPU load and processing speed.  
-' defval='0' minval='0' maxval='32'/>
+between CPU load and processing speed.  For Debian, the default it 1.
+' defval='1' minval='0' maxval='32'/>
     <option type='string' id='DOT_FONTNAME' format='string' docs='
 By default doxygen will use the Helvetica font for all dot files that 
 doxygen generates. When you want a differently looking font you can specify 
Index: doxygen-1.7.6.1/src/configoptions.cpp
===================================================================
--- doxygen-1.7.6.1.orig/src/configoptions.cpp	2011-12-12 14:26:13.867594965 +0000
+++ doxygen-1.7.6.1/src/configoptions.cpp	2011-12-12 14:29:54.897920578 +0000
@@ -2181,11 +2181,11 @@
   ci = cfg->addInt(
                  "DOT_NUM_THREADS",
                  "The DOT_NUM_THREADS specifies the number of dot invocations doxygen is\n"
-                 "allowed to run in parallel. When set to 0 (the default) doxygen will\n"
+                 "allowed to run in parallel. When set to 0 (the upstream default) doxygen will\n"
                  "base this on the number of processors available in the system. You can set it\n"
                  "explicitly to a value larger than 0 to get control over the balance\n"
-                 "between CPU load and processing speed.",
-                 0,32,0
+                 "between CPU load and processing speed. For Debian the default is 1",
+                 0,32,1
                 );
   //----
   cs = cfg->addString(
