Index: doxygen/src/htmlgen.cpp
===================================================================
--- doxygen.orig/src/htmlgen.cpp	2013-11-09 19:18:19.000000000 +0100
+++ doxygen/src/htmlgen.cpp	2013-11-09 19:18:19.000000000 +0100
@@ -1560,6 +1560,7 @@
     fprintf(stderr,"Warning: Cannot open file %s for writing\n",fileName.data());
   }
 
+  if (Config_getBool("GENERATE_TREEVIEW") || Config_getBool("SEARCHENGINE"))
   {
     QFile f(dname+"/jquery.js");
     if (f.open(IO_WriteOnly))
