Index: doxygen-1.7.3/addon/doxywizard/Makefile.doxywizard
===================================================================
--- doxygen-1.7.3.orig/addon/doxywizard/Makefile.doxywizard	2011-01-03 23:25:53.368234069 +0000
+++ doxygen-1.7.3/addon/doxywizard/Makefile.doxywizard	2011-01-03 23:26:23.118228985 +0000
@@ -16,14 +16,14 @@
 CXXFLAGS = -pipe -g -D_LARGEFILE_SOURCE -Wall -W -O2 -D_REENTRANT  -DQT_NO_CAST_FROM_ASCII -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_THREAD_SUPPORT -DQT_SHARED -DQT_TABLET_SUPPORT
 LEXFLAGS = 
 YACCFLAGS= -d
-INCPATH  = -I/usr/share/qt3/mkspecs/default -I. -I. -I/usr/include/qt3 -Imoc/
+INCPATH  = -I/usr/share/qt4/mkspecs/default -I. -I. -I/usr/include/qt4 -Imoc/
 LINK     = g++
 LFLAGS   = 
-LIBS     = $(SUBLIBS) -L/usr/share/qt3/lib -L/usr/X11R6/lib -lqt-mt -lXext -lX11 -lm -lpthread
+LIBS     = $(SUBLIBS) -L/usr/share/qt4/lib -L/usr/X11R6/lib -lqt-mt -lXext -lX11 -lm -lpthread
 AR       = ar cqs
 RANLIB   = 
 MOC      = moc
-UIC      = /usr/share/qt3/bin/uic
+UIC      = /usr/share/qt4/bin/uic
 QMAKE    = qmake
 TAR      = tar -cf
 GZIP     = gzip -9f
